### PR TITLE
Feature/build version

### PR DESCRIPTION
**Pull request by Paulina-Grochal**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,6 @@ bin/
 
 # Goreleaser Output
 dist/
+
+# VSCode setup and plugins
+.vscode/
```
Comment (cloudlena) [inline .gitignore:19]: This should be part of a VS Code user's global `.gitignore` and, IMHO, has nothing to do with the project.

File: go.mod [hunk 2]
```diff
@@ -45,6 +45,7 @@ require (
 	github.com/muesli/termenv v0.12.0 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/rivo/uniseg v0.2.0 // indirect
+	github.com/ropes/go-linker-vars-example v0.0.0-20151228090247-4d2722ea9b87 // indirect
```
Comment (cloudlena) [inline go.mod:48]: Is this change intended?

File: Makefile [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+GITHASH := $(shell git rev-list -1 HEAD)
+
+LDFLAGS=-ldflags="-w -s -X 'main.buildversion=${BUILDVERSION}' -X 'main.builddate=${BUILDDATE}' -X 'main.githash=${GITHASH}'"
+
 .PHONY: build
 all:
-	go build -o bin/livelint
+	go build -o ${BUILDDIR}
+
+.PHONY: build_ldflags
```
Comment (cloudlena) [inline Makefile:12]: This is incomplete (the `.PHONY` statement always needs to be the same as its following command).

I would not make this a new `make` command, but rather just change the `all` one to always include the build flags.

**Review by cloudlena — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (Paulina-Grochal) [reply Makefile:12]: ok let's leave everything in one step

**Review by cloudlena — APPROVED** [review]
